pypsa_to_IPCC.py

```diff
@@ -104,14 +104,17 @@
     #one excel file per scenario
     file = openpyxl.load_workbook(template_path)
     ds = file['data'] #data sheet
+    ratios = pd.read_csv("resources/industry_sector_ratios.csv",index_col=0,header=0,)
+
     for year in years:
         n = pypsa.Network(f"postnetworks/{scenario}{year}.nc")
         costs = pd.read_csv(f"costs/costs_{year}.csv", index_col=[0,1])
-        
+        prod = pd.read_csv("resources/industrial_production_elec_s_37_{}.csv".format(year),index_col=0,header=0,)  #s_37 ?? 
+        industry_demand=pd.read_table('resources/industrial_energy_demand_elec_s370_37m_{}.csv'.format(year),delimiter=',',index_col=0)
```

Comment:
the part in the file name 's370_37m' should not be hard-codded but read from the config file

Reply:
Also fixed.

Reply:
fixed in https://github.com/martavp/pypsa-eur-sec-to-ipcc/pull/33/commits/b940cb1a9c69d8ef9dbd2d23f0154a206d8fe54b . Thanks